app/js/components/drugOrderEntry/addForm/AddForm.jsx

```diff
@@ -81,6 +83,16 @@ export class AddForm extends React.Component {
     this.setState({ formType: formType.trim() });
   }
 
+  checkIfDrugHasActiveOrder = (drug) => {
+    const { filteredOrders } = this.props.fetchOrdersReducer;
+    const drugOrders = filteredOrders.filter(order => order.type === 'drugorder');
+    const findDrug = drugOrders.filter(order => order.drug.display === drug);
```

Comment:
Why are you using the`drug display` name to determine whether the drug exists? I think there are unique Identifiers that serve this purpose better.

Reply:
Will refactor to use uuid

Reply:
Done